src/plugin/yjsEditor.ts

```diff
@@ -99,15 +99,18 @@ function applyRemoteYjsEvents(editor: YjsEditor, events: Y.YEvent[]): void {
 
 export function withYjs<T extends Editor>(
   editor: T,
-  sharedType: SharedType
+  sharedType: SharedType,
+  { synchronizeValue }: WithYjsOptions = { synchronizeValue: true }
```

Comment:
```suggestion
  { synchronizeValue = true }: WithYjsOptions = {}
```

Reply:
otherwise `withYjs(editor, {})` would behave differently compared to `withYjs(editor)`

Reply:
Good catch.